client/src/components/TaskCard/TaskCard.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { PropTypes } from "prop-types";
+import SmallActionButton from "../SmallActionButton/SmallActionButton.js";
+import styles from "./TaskCard.module.css";
+
+const TaskCard = ({
+  title,
+  locationText,
```

Comment:
For props such as these, since we know have the PropTypes indicating that we'll require a string, we won't need `text` after the prop name. (i.e. `locationText`, `startTimeText` and `endTimeText` can just be `location`, `startTime` and `endTime`)

Reply:
Renamed each props accordingly